x-pack/plugins/maps/public/classes/styles/vector/style_util.ts

```diff
@@ -107,3 +109,10 @@ export function makeMbClampedNumberExpression({
     fallback,
   ];
 }
+
+export function getHasLabel(label: StaticTextProperty | DynamicTextProperty) {
```

Comment:
nit: `hasLabel` reads better imho

Reply:
agreed that `hasLabel` reads better, but then what do you call the results? I had to switch to `getHasLabel` so results could be called `hasLabel` without any naming conflict